src/app/gallery-page-component/gallery-page-component.ts

```diff
@@ -1,12 +1,26 @@
-import { Component } from "@angular/core";
+    import { Component } from "@angular/core";
+    import {  GalleryService } from "../services/gallery-service";
+    @Component({
+        selector:"my-gallery-component",
+        templateUrl:"./gallery-page-component.html",
+        styleUrls:["./gallery-page-component.css","../../assets/css/lightbox.min.css"]
+        
+    })
 
-@Component({
-    selector:"my-gallery-component",
-    templateUrl:"./gallery-page-component.html",
-    styleUrls:["./gallery-page-component.css","../../css/lightbox.min.css"]
-})
-
-export class MyGalleryPageComponent  {
-
-    
-}   
\ No newline at end of file
+    export class MyGalleryPageComponent  {
+        
+        gallery = []; 
+        labels =["all","indoor","outdoor","gd","babies","monthlyactivty","specialevent","birhday","schoolreadlines"];
```

Comment:
Labels should come from constant files
incase if we want to change it then we can see the constant file only